pom.xml

```diff
@@ -813,11 +813,9 @@
     </profile>
     <profile>
       <id>community</id>
-      <!-- jakarta TODO package javax.swing is declared in module java.desktop, but module jakarta.annotation does not read it etc
-      <modules>
+      <modules> 
         <module>narayana-full</module>
       </modules>
-      -->
```

Comment:
Could you please squeeze this commit with the previous one? Thanks

Reply:
Thanks @jmfinelli for the review. Yes, I will do it.